src/main/java/org/springframework/cloud/bindings/boot/OracleBindingsPropertiesProcessor.java

```diff
@@ -49,7 +49,7 @@ public void process(Environment environment, Bindings bindings, Map<String, Obje
             map.from("username").to("spring.datasource.username");
             map.from("password").to("spring.datasource.password");
             map.from("host", "port", "database").to("spring.datasource.url",
-                    (host, port, database) -> String.format("jdbc:oracle://%s:%s/%s", host, port, database));
+                    (host, port, database) -> String.format("jdbc:oracle:thin:@%s:%s/%s", host, port, database));
```

Comment:
As I commented in https://github.com/spring-cloud/spring-cloud-bindings/issues/82#issuecomment-1263078486, this is incomplete

Reply:
It is incomplete. But the old code doesn't work in any scenario. I just changed it to work in certain scenarios. 
It is not possible to cover all the scenarios. Users will have to use `jdbc-url` in these cases.

If it is necessary to cover the three connection types (i.e. SID, Service Name, TNS), which [key](https://github.com/spring-cloud/spring-cloud-bindings#oracle-rdbms) should the user use to specify the connection type? 
If the users don't specify the connection type via service binding, the code will not be able to determine what is the connection type.

Reply:
> `jdbc:oracle:thin:@%s:%s/%s`

Did you mean `jdbc:oracle:thin:@//%s:%s/%s` (service name) or `jdbc:oracle:thin:@%s:%s:%s` (sid) ?

I think it's good to introduce new keys `connection-type` (default sevice name or sid) and `driver` (default thin)

Reply:
I have updated the code to cover scenarios for sid, service name and tns. Please take a look. Thanks.